trp_project_scrum/security/res_groups.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="external_scrum_users" model="res.groups">
```

Comment:
please add both of those to the external users category as in https://github.com/Therp/Therp-Addons/blob/10.0/trp_external_user/security/res_groups.xml#L5